programs/balanced-dollar/src/instructions.rs

```diff
@@ -51,11 +51,26 @@ pub fn cross_transfer<'info>(
     ctx: Context<'_, '_, '_, 'info, CrossTransfer<'info>>,
     to: String,
     value: u64,
```

Comment:
Do we even need this when we have ctx.accounts.from.amount?

Reply:
this is the total amount user hold,  was thinking, there should be limit how much use want to spend?

Reply:
Yeah okey, we could potentially still opt to only have ICON_bnUSD_value

Reply:
yeah done